cli/cli/commands/files/upload/upload.go

```diff
@@ -97,15 +99,3 @@ func run(
 	logrus.Infof("Files package '%v' uploaded with UUID: %v", fileArtifactName, filesArtifactUuid)
 	return nil
 }
-
-func validatePathArg(_ context.Context, _ *flags.ParsedFlags, args *args.ParsedArgs) error {
```

Comment:
Are we removing this because the filepath arg already does this?

Reply:
> Are we removing this because the filepath arg already does this?

Yes